File: 250623/문자열의 특정 위치 찾기 2/find-specific-location-in-spring-2.py
arr = ["apple", "banana", "grape", "blueberry", "orange"]

a = input()
cnt = 0

for i in range(5):
    for j in range(2,4):
        if arr[i][j] == a:
            print(arr[i])
            cnt += 1
            continue

print(cnt)

File: 250709/문자열의 특정 위치 찾기 2/find-specific-location-in-spring-2.c
#include <stdio.h>
#include <string.h>

int main() {
    char* fruit[5] = {"apple", "banana", "grape", "blueberry", "orange"};
    char n;
    int num=0;
    
    scanf("%c", &n);
    
    for (int i=0; i<5; i++){
        if (fruit[i][2]==n){
            num++;
            printf("%s\n", fruit[i]);
        }
        else if (fruit[i][3]==n){
            num++;
            printf("%s\n", fruit[i]);
        }
    }
    printf("%d", num);
    return 0;
}